metrics/stats.go

```diff
@@ -155,7 +186,16 @@ func GetStats() *Stats {
 		GcPerSecond:      gcPerSecond,
 		GcPausePerSecond: gcPausePerSecond,
 		GcPause:          gcPause,
-		// Connections
-		Connections: GetConnection(),
+	}
+}
+
+func GetPlasmaStats() *PlasmaStats {
```

Comment:
__[golint]__
* [golint] exported function GetPlasmaStats should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->